libs/solid/skeleton/src/lib/services/feedback.service.ts

```diff
@@ -23,6 +24,13 @@ export function feedbackServiceFactory(
 }
 
 export class FeedbackService {
+  /**
+   * use this variable to save the value of the component's url 
+   * 1. if the user open it from the toolbar => the url will be "menu"
+   * 2. if the user open it from the sidebar(Menu) => the url will be the current's url 
+   */
+  private currentUrl : string = ""; 
```

Comment:
Hier gibt mein Linter eine Fehlermeldung aus: 
`Type string trivially inferred from a string literal, remove type annotation.`

(siehe auch dieser [Stackoverflow-Thread](https://stackoverflow.com/questions/46799828/tslint-type-trivially-inferred-why-is-it-bad-practice-to-include-the-type-he))

Reply:
Das müsste dann sein
```
private currentUrl = '',
```